taskcluster/win-amd64-gpu-opt.yml

```diff
@@ -6,6 +6,8 @@ build:
     - "index.project.deepspeech.deepspeech.native_client.win-cuda.${event.head.sha}"
     - "notify.irc-channel.${notifications.irc}.on-exception"
     - "notify.irc-channel.${notifications.irc}.on-failed"
+  dependencies:
+    - "node-gyp-cache"
```

Comment:
Why do these tasks depend on node-gyp-cache? They should just get the artifacts from the indexed old task, no? Otherwise this is effectively reaching the ElectronJS servers every time anyway?

Reply:
No, this makes the task cache transparent: logic inside will check if the artifact exists. If it does it re-copies it (I don't know if it's a bug or a feature of the taskcluster index, but if we resolve a task with an exisiting index and that this task has no artifact, then we loose artifacts), and if the artifact does not exists, it will create it

Reply:
Yep, makes total sense, I missed this behavior when checking. Should we remove the date number from the index then? Seems misleading.

Reply:
> Yep, makes total sense, I missed this behavior when checking. Should we remove the date number from the index then? Seems misleading.

That's a simple way to invalidate the cache, if you can think of a better one I'd be happy to switch it

Reply:
I was thinking an increasing integer value for example, since this won't be connected to the date after the artifacts expire for the first time and get re-created.

Reply:
Moved to an integer and added reference in bash script so that we don't forget to update.